.changeset/ten-actors-mate.md

```diff
@@ -0,0 +1,5 @@
+---
+'@lowdefy/docs': patch
+---
+
+fix broken link and use improved reference to Google Docs
```

Comment:
`Fix broken link and use improved reference to Google Docs.`

Reply:
Done - 9f29bb0645b59eb9a2958e6e3e1aaefb9ba0cea9